src/lib/translators/mongo.ts

```diff
@@ -588,9 +588,13 @@ const mapper: Partial<StepMatcher<MongoStep>> = {
       },
     },
   }),
-  delete: (step: Readonly<S.DeleteStep>) => ({
-    $project: _.fromPairs(step.columns.map(col => [col, 0])),
-  }),
+
+  delete(this: Mongo36Translator, step: Readonly<S.DeleteStep>) {
+    return {
+      $project: _.fromPairs(step.columns.map(col => [this.escapeFieldName(col), 0])),
```

Comment:
`this.escapeFieldName` is very clear but quite long to write.
Does someone have a better naming idea? I really like the `$$` shortcut.

Reply:
`$$` is already used in our codebase (but maybe that's what you meant). `escapeField`, `quoteField`, `$field`, `$quote`, `field$`, `quote$` `escapeIdentifier`, `quoteIdentifier` or `quoteId` seems fine to me :)

We could even do things a little differently with something like this:

```typescript
function Field(name) { this.name = name; }
function $field(name) { return new Field(name) }
function isField(obj: any) : obj is Field{ return obj instanceof Field; }
```

and afterwards use `$field` everywhere explicitly and have a general postprocess visitor that would do the appropriate escaping. 

General question: where exactly should these fields be escaped? Only when used as keys or also when used as values?

Reply:
@adimasci That was only for keys
But after discussions I has with @vdestraitt and @rhuille, we thought it may be better not to do this at the moment. (See internal slack thread: https://toucantoco.slack.com/archives/C08DRSQ5U/p1582819058152100)